config.yaml

```diff
@@ -0,0 +1,18 @@
+linkerdVersion: stable-2.8.0
+externalIssuer: false
+globalControlPlane:
+    enable: true
+    uninstall: true
+addOns:
+    tracing:
+        enabled: true
+        collector:
+            resources:
+                cpu:
+                    limit: 100m
+                    request: 10m
+                memory:
+                    limit: 100Mi
+                    request: 50Mi:W
```

Comment:
typo ;)

Reply:
Oops! If only I could be a little more careful while using vim..